classifiers/python_functions/lookup_list/code_snippet.md

```diff
@@ -1,14 +1,15 @@
 ```python
 # the lookup list values can also come from a knowledge base, e.g.:
 # from knowledge import my_lookup_list
+from typing import List
 
-YOUR_ATTRIBUTE = "sender"
-YOUR_LOOKUP_LIST = ["johannes.hoetter@kern.ai", "henrik.wenck@kern.ai"]
-YOUR_LABEL = "ham"
+YOUR_ATTRIBUTE: str = "text"
+YOUR_LOOKUP_LIST: List[str] = ["johannes.hoetter@kern.ai", "henrik.wenck@kern.ai"]
```

Comment:
is this supposed to be a list of values the user can enter or a lookuplist we use in the application?
a list of strings would be something like YOUR_VALUES_TO_ITERATE: List[str].

for a lookuplist form refinery we need to import knowledge and have something like
YOUR_LOOKUP_LIST: str = clickbait_expression 
this however would be only a str in regards to the logic and a list later in the code.

Reply:
As the module name suggests, we should have a lookup list when running the code in refinery: so I would go with the latter in that case.

Reply:
changed that both is possible as with other lookup instances